packages/javascript/bh-shared-ui/src/views/TierManagement/Details/MembersList.tsx

```diff
@@ -99,11 +101,18 @@ const getFetchCallback = (selectedTier: number, selectedSelector: number | null)
     }
 };
 
+const getListHeight = (windoHeight: number) => {
+    if (windoHeight > 1080) return 762;
+    if (1080 >= windoHeight && windoHeight > 900) return 642;
+    if (900 >= windoHeight) return 438;
+    return 438;
+};
+
 interface MembersListProps {
-    selectedTag: number;
-    selectedSelector: number | null;
-    selected: number | null;
-    onClick: (id: number, data: AssetGroupTagSelectorNode) => void;
+    selectedTag: string | undefined;
+    selectedSelector: string | undefined;
+    selected: string | undefined;
```

Comment:
For anyone reviewing and wondering why `key: string | undefined;` versus `key?: string;`, see below.

```tsx
interface CompAProps {
  value: string | undefined;
}

// Works
<CompA value="" />

// Works
<CompA value={undefined} />

// Type error
<CompA />


interface CompBProps {
  value?: string;
}

// Works
<CompB value="" />

// Type error
<CompB value={undefined} />

// Works
<CompA />
```